src/main/java/core/basesyntax/strategy/BalanceOperationStrategy.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.strategy;
+
+public class BalanceOperationStrategy implements OperationStrategy {
+    @Override
+    public int calculate(final int number) {
+        return number;
+    }
+}
```

Comment:
```suggestion
public class BalanceOperationHandler implements OperationHandler {
    @Override
    public void handle(FruitTransaction transaction) {
        Storage.storage.put(transaction.getFruit(), transaction.getAmount());
    }
}
```
this class should be called Handler, and it should make changes to storage (let's create a class Storage, that is those imitate a db)

Reply:
Done